src/Umbraco.Web.BackOffice/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -37,6 +38,15 @@ public static void AddUmbracoBackOfficeIdentity(this IServiceCollection services
             services.ConfigureOptions<ConfigureBackOfficeSecurityStampValidatorOptions>();
         }
 
+        /// <summary>
+        /// Adds the services required for using Members Identity
+        /// </summary>
+        public static void AddMembersIdentity(this IServiceCollection services) =>
```

Comment:
This can't be in the BackOffice assembly because it will need to be used on the front-end. Since it will be used in both it needs to be part of "Common"

Reply:
Good point, will move

Reply:
Moved to common